autowarefoundation/autoware
# add map_extender

## Status
**DEVELOPMENT**

## Description
Added map_extender in map_tools package

## Todos


## Steps to Test or Reproduce
1. Load points_map and corresponding TF in Map tab.
2. Open RViz and open config map_extender.rviz.
3. rosrun map_tools map_extender "pcd file"
4. Specify the connecting point between the base map and the new map using 2D Pose Estimate in RViz.
5. Extended map will be appear in the same directory as the input pcd file.
